arch/x86/soc/ia32/soc.c

```diff
@@ -26,4 +26,7 @@ MMU_BOOT_REGION(CONFIG_IOAPIC_BASE_ADDRESS, MB(1), MMU_ENTRY_WRITE);
 #ifdef CONFIG_HPET_TIMER
 MMU_BOOT_REGION(CONFIG_HPET_TIMER_BASE_ADDRESS, KB(4), MMU_ENTRY_WRITE);
 #endif
+
+MMU_BOOT_REGION(0xfebc0000, KB(128), MMU_ENTRY_WRITE);
```

Comment:
What is this this address? have CONFIG_ for it.

Reply:
Fixed.